frontend/smart-kickers-game/src/components/Button.css

```diff
@@ -0,0 +1,49 @@
+:root {
+  --primary: rgba(212, 238, 194, 0.822);
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
This is a global style that will apply to all `a` elements the application. Either replace it with something more specific (like class selector), if it should be applied only for your `<Button>`s, or move it to a global css file at the application root.